src/domains/idpay-common/04_cdn.tf

```diff
@@ -49,7 +49,7 @@ locals {
  */
 // public storage used to serve FE
 module "idpay_cdn" {
-  source = "git::https://github.com/pagopa/terraform-azurerm-v3.git//cdn?ref=v6.18.0"
+  source = "git::https://github.com/pagopa/terraform-azurerm-v3.git//cdn?ref=f9ea700"
```

Comment:
is it wanted the different ref at cdn module? one for sha and one for version?

Reply:
it is temporary. i was waiting for the pr to be approved on the module, so that the new tag is available